istio/scripts/loadgen.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+export INGRESS_NAME=istio-ingress
+export INGRESS_NS=istio-ingress
+export INGRESS_HOST=$(kubectl -n "$INGRESS_NS" get service "$INGRESS_NAME" -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+export INGRESS_PORT=$(kubectl -n "$INGRESS_NS" get service "$INGRESS_NAME" -o jsonpath='{.spec.ports[?(@.name=="http2")].port}')
+export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+
+echo "Generating Load"
```

Comment:
I asked this on the Solr sample app as well. Would using k6 improve the load testing at all over just using curl?

Reply:
curl seems to be good enough right now as far as I can tell

Reply:
Cool. I tend to agree, just had to ask the question in case there was any probable benefit. :)